backend/api/Models/Concepts/Property/PropertyMap.cs

```diff
@@ -32,6 +32,8 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.PphStatusUpdateTimestamp, src => src.PphStatusUpdateTimestamp)
                 .Map(dest => dest.PphStatusUpdateUserGuid, src => src.PphStatusUpdateUserGuid)
                 .Map(dest => dest.Notes, src => src.Notes)
+                .Map(dest => dest.IsOwned, src => src.IsOwned)
+                .Map(dest => dest.IsVisibleToOtherAgencies, src => src.IsVisibleToOtherAgencies)
```

Comment:
Is this field `IsVisibleToOtherAgencies` relevant to this implementation of PIMS? Seems something like from legacy CITZ PIMS...